media/css/firefox/whatsnew/whatsnew-144beta.scss

```diff
@@ -0,0 +1,49 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import 'includes/base';
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/notification-bar';
+@import 'includes/dark-mode';
+@import 'includes/mofo-donate-cta';
+
+.wnp-content-main {
+    p {
+      margin-inline: 0;
+      margin-block-end: 2em;
+    }
+
+    @media (prefers-color-scheme: dark) {
+        a:link,
+        a:visited {
+            color: white;
+        }
+    }
+}
+
+h1, .wnp-content-cta {
+    text-align: center;
+}
+
+h1 {
+    @include text-title-md;
+}
+
+.wnp-footer {
+    text-align: center;
+}
+
+.wnp-content-img {
+    display: block;
+    margin-block-end: 0.5em;
+    margin-inline: auto;
+
+    @media ($mq-sm) {
+        float: left;
+        height: 8lh;
```

Comment:
Manipulated height a bit to avoid a weird dangling line with full-width
<img width="786" height="505" alt="Screenshot 2025-09-16 at 8 04 17 AM" src="https://github.com/user-attachments/assets/2578bb63-94c9-41ad-8582-8faeca7a6c60" />

Reply:
🙇  TIL as usually